pandas_gbq/gbq.py

```diff
@@ -727,6 +740,21 @@ def _localize_df(schema_fields, df):
     return df
 
 
+def _make_bqstorage_client(use_bqstorage_api, credentials):
```

Comment:
Should this take a project too?

Reply:
The BigQuery Storage API client doesn't actually take a project. The project is set when creating a BQ Storage API read session by `to_dataframe` based on the BQ client's project.

Aside: The BQ Storage API client is partially auto-generated, and the client generator doesn't support a default project ID. Since I'm considering the BQ Storage API a lower-level client that's used by the normal BQ client from `to_dataframe`, I felt it wasn't worth adding in that kind of handwritten helper to the autogenned layer.